### PR TITLE
[WIP] Fix W&B Artifacts Integration and minor errors

### PR DESCRIPTION
**Pull request by digantamisra98**
Fixing some errors with the current artifacts integration.

### CODE REVIEW

**Comment by digantamisra98** [comment]
It seems that the `copy.deepcopy(list(strategy.model.parameters()))` does work but returns the params as a list cloned from the source generator object in the `evaluation/metrics/checkpoint.py` which when used in `logging/wandb_logger.py` doesn't seem to pass through the `if isinstance(value, Tensor):` line since it demands a Tensor while the metric returns a List.

File: examples/wandb_logger.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
 from os.path import expanduser
 
 import argparse
+
+from click import Path
```
Comment (AndreaCossu) [inline examples/wandb_logger.py:25]: is this import used somewhere?

Comment (digantamisra98) [reply examples/wandb_logger.py:25]: Oh, my bad, added it by mistake.

**Comment by AndreaCossu** [comment]
@digantamisra98 any news on this?

**Comment by digantamisra98** [comment]
@AndreaCossu I am no longer at W&B. But I am happy to finish this but won't be able to before December. Thanks!

**Comment by lrzpellegrini** [comment]
Added in #1463

**Pull request closed without merging**